packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -103,6 +103,12 @@ export interface IContextualMenuProps extends React.Props<ContextualMenu> {
   onDismiss?: (ev?: any, dismissAll?: boolean) => void;
 
   /**
+   * Click handler which is invoked if onClick is not passed for individual contextual
+   * menu item
+   */
+  onItemClick?: (ev?: React.MouseEvent<HTMLElement>, item?: IContextualMenuItem) => void;
```

Comment:
Yep, I skimmed through the docs and most of the onSometing handlers are passing item first (not all of the though). I'll refactor it